test/Test/Pos/Block/Logic/VarSpec.hs

```diff
@@ -34,6 +40,8 @@ spec = describe "Block.Logic.VAR" $ modifyMaxSuccess (min 12) $ do
     describe "verifyBlocksPrefix" verifyBlocksPrefixSpec
     describe "verifyAndApplyBlocks" verifyAndApplyBlocksSpec
     describe "applyBlocks" applyBlocksSpec
+    describe "Block.Event" $ do
```

Comment:
Because it's disabled in hlint. Humans know better. We have many such `do`s.

In general I don't like redundant `do`s, but in `hspec` it's quite natural.